apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -90,30 +81,32 @@ const ScrollButtons = React.createClass({
 
     // If mouse is held down for half a second, begin gradual continuous
     // scroll
-    var timeout = setTimeout(function () {
-      var interval = setInterval(function () {
+    var scrollTimeout = setTimeout(function () {
+      var scrollInterval = setInterval(function () {
         let dist = CONTINUOUS_SCROLL_BY;
         if (dir === DIRECTIONS.UP) {
           dist *= -1;
         }
         scrollBy(contentContainer, dist, false);
       }.bind(this), CONTINUOUS_SCROLL_INTERVAL);
-      this.setState({
-        scrollInterval: interval
-      });
+      this.setState({ scrollInterval });
     }.bind(this), CONTINUOUS_SCROLL_TIMEOUT);
 
+    const unbindMouseUp = addMouseUpTouchEvent(document, this.scrollStop);
     this.setState({
-      scrollTimeout: timeout
+      scrollTimeout,
+      unbindMouseUp
```

Comment:
Could/should `unbindMouseup` (as well as `scrollTimeout` and `scrollInterval`) live directly on `this` instead of being part of `state`?